*** auto-tests/A-05-04-M-ok.out	2020-10-17 17:23:28.457373000 +0100
--- auto-tests/A-05-04-M-ok.outhyp	2020-12-10 17:57:23.520584000 +0000
***************
*** 38,44 ****
  4 - Activa/desactivar notificações de um produto
  5 - Mostrar transacções do cliente
  0 - Sair
! Escolha uma opção: Identificador do cliente: Nome do cliente: Endereço do cliente: Registar cliente: Operação inválida: O cliente '2' já existe.
  Gestão de Clientes
  1 - Mostrar cliente
  2 - Mostrar clientes
--- 38,44 ----
  4 - Activa/desactivar notificações de um produto
  5 - Mostrar transacções do cliente
  0 - Sair
! Escolha uma opção: Identificador do cliente: Nome do cliente: Endereço do cliente: O cliente '2' já existe.
  Gestão de Clientes
  1 - Mostrar cliente
  2 - Mostrar clientes
